lib/all/index.js

```diff
@@ -1,6 +1,10 @@
 import curry2 from '../_internal/_curry2'
-import toArray from '../toArray'
 
 export default curry2(function all(cb, data) {
-  return toArray(data).every(cb)
+  var i = 0
+  var length = data.length
+  while (i < length && cb(data[i])) {
+    i += 1
```

Comment:
why not `cb(data[i++])` or `i++` inside `while` block?

Reply:
Because, I've tested it and [it's slower](https://github.com/nanoutils/nanoutils/pull/102#discussion_r183655882)

Reply:
@BabkinAleksandr agree, thanks
missed the point above